JYH/3Day/K번째수.js

```diff
@@ -0,0 +1,13 @@
+function solution(array, commands) {
+    var answer = [];
+    
+    for (let i = 0; i < commands.length; i++) {
+        let start = array.slice(commands[i][0] - 1, commands[i][1])
+        start.sort(function(a, b) {
+            return a - b;
+        })
```

Comment:
```suggestion
        start.sort((a, b) => a - b)
```
화살표 함수를 쓰면 조금 더 간단하게 표현가능한 점 알아두시면 좋을 것 같습니다!

Reply:
조금 더 찾아보고 배워서 바꾸도록 하겠습니다 감사합니다!